src/pages/profile/index.js

```diff
@@ -129,6 +147,7 @@ const ViewProfile = () => {
                     className="textarea-small"
                     placeholder="Cohort 4"
                     disabled
+                    value={`Cohort ` + user.cohortId}
```

Comment:
The cohort ID might not always match the cohort number/name - is there a different property you can use?

This might mean needing to make an additional request to get cohort data if there's not enough data in the request to get the user data.

Reply:
Additionally, please extract this to a utility function - for now in the component. 

so this should look like:
```js
value={cohortDisplayName(cohort)}
```
And the function can be responsible for how it should display

Reply:
> The cohort ID might not always match the cohort number/name - is there a different property you can use?

As far as I can see in the API docs, this is the only ID/name related to cohorts and I can't see any other properties that might be useful, unless we're missing something?